lib/Db/Feed.php

```diff
@@ -83,6 +83,8 @@ class Feed extends Entity implements IAPI, \JsonSerializable
     protected $basicAuthUser = '';
     /** @var string|null */
     protected $basicAuthPassword = '';
+    /** @var string|null */
+    protected $nextUpdateTime = '';
```

Comment:
I don't think it makes a lot of sense to have a timestamp as a string 
```suggestion
    /** @var int|null */
    protected ?int $nextUpdateTime = null;
```

Reply:
Copilot thought so too and I would agree but I checked what was already there and the lastmodified is a string, added is an int, httplastmodified is a string and stored as formated date in the db...

I'm fine adding it as int though.
I think at some time some chore work would be nice for news to adjust variable types and also the naming